ephemeris/shed_install.py

```diff
@@ -229,29 +129,34 @@ def installed_tool_revisions(gi=None, omit=None):
     """
     if not omit:
         omit = []
-    tsc = tool_shed_client(gi)
+    tool_shed_client = ToolShedClient(gi)
+
+    # Create dictionary to look up all tools based on repository information
+
     installed_revisions_list = []
-    itl = tsc.get_repositories()
-    for it in itl:
-        if it['status'] == 'Installed':
+    installed_tool_list = tool_shed_client.get_repositories()
```

Comment:
Now that you're at it, can we change this to `installed_repository_list`, to avoid any confusion, and to leave some wigglespace should we ever implement a more tool-centric installation approach?

In fact I think you can s/tool/repository/g and s/tools/repositories/g in this file, this should be much clearer.

Reply:
On my way